docs/index.html

```diff
@@ -1,14 +1,18 @@
 <!DOCTYPE HTML>
 <html lang="en-US">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="refresh" content="0; url=https://agentos.org/latest/">
-        <script type="text/javascript">
-            window.location.href = "https://agentos.org/latest/"
-        </script>
-        <title>Page Redirection</title>
-    </head>
-    <body>
-        If you are not redirected automatically, click to visit the <a href='https://agentos.org/latest/'>most recent agentos docs</a>.
-    </body>
-</html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="refresh" content="0; url=https://agentos.org/latest/">
+    <script type="text/javascript">
+        window.location.href = "https://agentos.org/latest/"
+    </script>
+    <title>AgentOS: a learning agent platform</title>
+</head>
+
+<body>
+    Open Source python API & CLI for developing learning agents. This page should redirect to the latest AgentOS
```

Comment:
While we're making changes anyway, can you please line wrap at <=79 characters? As of https://github.com/agentos-project/agentos/commit/d7d52f288ae6b324157e2d0d2f5e8a11d4316cbf we are doing that in the python code now, so might as well do it in the html code too.

Reply:
I will also line wrap like you mentioned.